pkgs/applications/networking/mailreaders/evolution/evolution-ews/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchurl, gnome3, cmake, gettext, intltool, pkg-config, evolution-data-server
+, sqlite, gtk3, webkitgtk, libgdata }:
+
+stdenv.mkDerivation rec {
+  pname = "evolution-ews";
+  version = "3.36.4";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/evolution-ews/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "0zfq02h3r1qbxak04i49564q4s2ykvkgcyc3krjgndan9lq3kvvn";
+  };
+
+  PKG_CONFIG_EVOLUTION_SHELL_3_0_ERRORDIR = "${placeholder "out"}/share/evolution/errors";
+  PKG_CONFIG_EVOLUTION_SHELL_3_0_PRIVLIBDIR = "${placeholder "out"}/lib/evolution";
+  PKG_CONFIG_CAMEL_1_2_CAMEL_PROVIDERDIR = "${placeholder "out"}/lib/evolution-data-server/camel-providers";
+  PKG_CONFIG_LIBEDATA_BOOK_1_2_BACKENDDIR = "${placeholder "out"}/lib/evolution-data-server/addressbook-backends";
+  PKG_CONFIG_LIBEDATA_CAL_2_0_BACKENDDIR = "${placeholder "out"}/lib/evolution-data-server/calendar-backends";
+  PKG_CONFIG_LIBEBACKEND_1_2_MODULEDIR = "${placeholder "out"}/lib/evolution-data-server/registry-modules";
+  PKG_CONFIG_EVOLUTION_SHELL_3_0_MODULEDIR = "${placeholder "out"}/lib/evolution/modules";
+  PKG_CONFIG_EVOLUTION_DATA_SERVER_1_2_PRIVDATADIR = "${placeholder "out"}/share/evolution-data-server";
```

Comment:
Could you move this under `cmakeFlags`, I believe it fits there better as it also configures the build.

Reply:
It does not build anymore if I move them to cmakeFlags. I end back here: https://github.com/NixOS/nixpkgs/pull/91504#issuecomment-653476825 when I try to move them to cmakeFlags

Reply:
Sorry, I meant moving them below/after `cmakeFlags` in the code, not inside.

Reply:
oh, ok that's doable ;)

Reply:
done.